mozilla/kuma
# fix bug 792947 - sitemap.xml (also bug 780740)

To test:

```
./manage.py cron build_sitemap
```

Check media/sitemap.xml

Note: We'll file bugs with webops to add `Alias /sitemap.xml media/sitemap.xml`
